modules/engage-theodul-plugin-tab-transcript/src/main/resources/static/styles/desktop.css

```diff
@@ -0,0 +1,20 @@
+.highlight > p > span:hover {
```

Comment:
Add more specificity to the CSS classes, like make them all dependent on #engage_Transcript_tab 
`#engage_Transcript_tab .highlight > p > span:hover`

Otherwise this might introduce CSS bleed over